src/lib/skilling/skills/hunter/birdHouseTrapping.ts

```diff
@@ -178,4 +179,182 @@ const birdhouses: Birdhouse[] = [
 	}
 ];
 
+export const birdhouseSeeds = [
+	{
+		name: 'Hammerstone seed',
+		itemID: itemID('Hammerstone seed'),
+		amount: 10
+	},
+	{
+		name: 'Asgarnian seed',
+		itemID: itemID('Asgarnian seed'),
+		amount: 10
+	},
+	{
+		name: 'Barley seed',
+		itemID: itemID('Barley seed'),
+		amount: 10
+	},
+	{
+		name: 'Yanillian seed',
+		itemID: itemID('Yanillian seed'),
+		amount: 10
+	},
+	{
+		name: 'Krandorian seed',
+		itemID: itemID('Krandorian seed'),
+		amount: 10
+	},
+	{
+		name: 'Wildblood seed',
+		itemID: itemID('Wildblood seed'),
+		amount: 5
+	},
+	{
+		name: 'Potato seed',
+		itemID: itemID('Potato seed'),
+		amount: 10
+	},
+	{
+		name: 'Onion seed',
+		itemID: itemID('Onion seed'),
+		amount: 10
+	},
+	{
+		name: 'Cabbage seed',
+		itemID: itemID('Cabbage seed'),
+		amount: 10
+	},
+	{
+		name: 'Tomato seed',
+		itemID: itemID('Tomato seed'),
+		amount: 10
+	},
+	{
+		name: 'Sweetcorn seed',
+		itemID: itemID('Sweetcorn seed'),
+		amount: 10
+	},
+	{
+		name: 'Strawberry seed',
+		itemID: itemID('Strawberry seed'),
+		amount: 10
+	},
+	{
+		name: 'Watermelon seed',
+		itemID: itemID('Watermelon seed'),
+		amount: 10
+	},
+	{
+		name: 'Marigold seed',
+		itemID: itemID('Marigold seed'),
+		amount: 10
+	},
+	{
+		name: 'Rosemary seed',
+		itemID: itemID('Rosemary seed'),
+		amount: 10
+	},
+	{
+		name: 'Nasturtium seed',
+		itemID: itemID('Nasturtium seed'),
+		amount: 10
+	},
+	{
+		name: 'Woad seed',
+		itemID: itemID('Woad seed'),
+		amount: 10
+	},
+	{
+		name: 'Limpwurt seed',
+		itemID: itemID('Limpwurt seed'),
```

Comment:
Item name shouldn't be kept like this, manually written out (its hard to maintain and prone to mistakes). Instead, just do `item: getOSItem("Limpwurt seed")`. and then you get the item.name and item.id.

Reply:
Updated the code to use getOSItem and changed consumers to use `item.id`